code-challenges.test.js

```diff
@@ -15,20 +15,62 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// a describe method that lists the name of the function OR naming of the particular test.
+describe("fib", () => {
+
+    // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+    it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+  
+      //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+      expect(fib(6)).toEqual([1, 1, 2, 3, 5, 8])
+      expect(fib(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+  })
+
 // Example input: 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// got 2 failed
 
 // b) Create the function that makes the test pass.
 
+// create a function named fib 
+// takes in a number in parameter
+// create an array [1,1]
+// create a for loop
+// use Fibonacci sequence to get correct output 
+// return the function into an array
+
+function fib(number) {
+  const fibs = [1, 1];
+  for (let i = 2; i < number; i++) {
+      fibs.push(fibs[i - 1] + fibs[i - 2]);
+  }
+  return fibs;
+}
```

Comment:
Excellent work!  A for loop is an excellent approach to this problem.  What made you decide to use function notation instead of arrow notation?